components/husky_a300/husky_a300_battery_balancing_quick.mdx

```diff
@@ -0,0 +1,35 @@
+import Admonition from "@theme/Admonition";
+
+### Battery Cell Balancing _(Quick)_  {#quick-cell-balance}
```

Comment:
Can you review this section with JasonB? I think he had me remove it for the Inventus batteries.

Reply:
We reviewed in person. I have removed this file due to the discussion.